drivers/syslog/vsyslog.c

```diff
@@ -138,94 +140,74 @@ int nx_vsyslog(int priority, FAR const IPTR char *fmt, FAR va_list *ap)
   if (d_ret > 0)
     {
 #if defined(CONFIG_SYSLOG_TIMESTAMP_FORMAT_MICROSECOND)
-      ret += lib_sprintf(&stream.public, "[%s.%06ld] ",
-                         date_buf, ts.tv_nsec / NSEC_PER_USEC);
+      offset += sprintf(offset, "[%s.%06ld] ",
+                        date_buf, ts.tv_nsec / NSEC_PER_USEC);
 #else
-      ret += lib_sprintf(&stream.public, "[%s] ", date_buf);
+      offset += sprintf(offset, "[%s] ", date_buf);
 #endif
     }
 #else
-  ret += lib_sprintf(&stream.public, "[%5jd.%06ld] ",
-                     (uintmax_t)ts.tv_sec, ts.tv_nsec / NSEC_PER_USEC);
+  offset += sprintf(offset, "[%5jd.%06ld] ",
+                    (uintmax_t)ts.tv_sec, ts.tv_nsec / NSEC_PER_USEC);
 #endif
 #endif
 
 #if defined(CONFIG_SMP)
-  ret += lib_sprintf(&stream.public, "[CPU%d] ", up_cpu_index());
+  offset += sprintf(offset, "[CPU%d] ", up_cpu_index());
 #endif
 
 #if defined(CONFIG_SYSLOG_PROCESSID)
   /* Prepend the Thread ID */
 
-  ret += lib_sprintf(&stream.public, "[%2d] ", gettid());
+  offset += sprintf(offset, "[%2d] ", (int)gettid());
 #endif
 
 #if defined(CONFIG_SYSLOG_COLOR_OUTPUT)
   /* Set the terminal style according to message priority. */
 
-  switch (priority)
+  FAR const char * const color_mapping[] =
```

Comment:
renanme to g_priority_color and move before line 42

Reply:
done